src/App/App.tsx

```diff
@@ -1,14 +1,19 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
 import { Auth, Hub } from 'aws-amplify';
 import { useState, useEffect } from 'react';
 import { withAuthenticator } from 'aws-amplify-react';
 import Sidebar from '../Components/Sidebar';
 import Content from '../Components/Content';
 import Navigation from '../Navigation/Navigation';
+import StudentPicture from '../assets/images/images-1.png';
+
 import './App.scss';
 
 // Entry point of the Flitped App
 function App() {
    const [, setUser] = useState(null);
+   const [username, setUsername] = useState('');
```

Comment:
if `username` is not used, consider getting ride of it.
```suggestion
   const [, setUsername] = useState('');
```